CONTRIBUTING.md

```diff
@@ -80,9 +80,18 @@ The following project structure should be followed:
 ```
 
 ## Generating Authentication Token
-- Navigate to `https://github.com/login/oauth/authorize?client_id=<GITHUB_CLIENT_ID>`
+- Run the project locally, make sure the server is listening to requests
+- Open a new tab in the browser
+- Open the `Network` tab in Browsers Developer Tools
+- Make sure the `Preserve log` checkbox is selected
+    <br />
+    ![Preserve logs checkbox](https://user-images.githubusercontent.com/26569942/102855521-cea32880-444a-11eb-84e3-02ee6fa5b830.png)
+- Navigate to `https://github.com/login/oauth/authorize?client_id=<GITHUB_CLIENT_ID>` in the same tab
 - Authorize the application.
-- Copy the response cookie named `rds-session` from the redirected request from the API.
+- In the browsers Dev tools Network tab, look for `<HOST>/auth/github/callback?code=<SOME_AUTOGENERATED_CODE>` request. This is the response that actually sets the cookie on the website. If we are making the call from localhost, it will not be honoured by the browser, but we can manually copy the cookie from **Response Headers** and use it for our Postman and Swagger calls.
```

Comment:
Can we just change this line to:

- Copy the response cookie named `rds-session` from the redirected request `<HOST>/auth/github/callback?code=<SOME_AUTOGENERATED_CODE>`.


We already have mentioned about using the token as bearer token in non production environments.
The other instructions mentioned about opening tab, preserve log is something too explicit and can be removed IMO. 🤔

Reply:
Also, the cookie will be set to the requests from localhost for running it locally and on staging environment as the cookie's SameSite attribute is set to `None`.

Reply:
@ankurnarkhede The cookie will most likely be ignored because it's secure, so needs to be sent over https. Can you look into running localhost with https?

Reply:
For running UI locally, requests to staging environment will contain the auth cookie as the `samesite` is set to `None` and the secure attribute is not valid for the requests originating from localhost (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#Attributes)

Reply:
@ankurnarkhede I checked on localhost, the cookie was being returned with `secure` attribute

Reply:
The secure flag is set but the requests to localhost will have the cookie added as localhost is exempted in case the secure flag is set. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#Attributes

Am I missing anything? 🤔

Reply:
@ankurnarkhede Oh. Interesting! I see what you are saying now. Thanks

I wonder why Chrome was complaining about it then. @whyDontI Did you also see that warning message?

Will have to check again, thanks Ankur!